src/test/java/io/camunda/zeebe/bpmnassert/assertions/JobAssertTest.java

```diff
@@ -28,13 +31,12 @@
   public static final String ERROR_CODE = "error";
   public static final String ERROR_MSG = "error occurred";
 
-  private ZeebeEngine engine;
-
   // These tests are for testing assertions as well as examples for users
   @Nested
   class HappyPathTests {
     private ZeebeClient client;
     private RecordStreamSource recordStreamSource;
```

Comment:
Do we still need this field now?

Reply:
No, I am gonna do a refactor of the tests and get rid of fields that aren't used anymore. I just didn't want this PR to become too big.